bin/travis-build.bash

```diff
@@ -15,29 +14,19 @@ echo "CKAN branch: $latest_ckan_release_branch"
 git checkout $latest_ckan_release_branch
 python setup.py develop
 pip install -r requirements.txt --allow-all-external
-pip install -r dev-requirements.txt --allow-all-external
```

Comment:
Let's comment this out but leave it in incase we need it at a later date.

Reply:
Okay. Done.